### PR TITLE
Ref#43698 [Admin] Manage list user

### PR DESCRIPTION
**Pull request by hoangtrungthong**
## Related Tickets:
- [#43649](https://edu-redmine.sun-asterisk.vn/issues/43649)

## WHAT this PR does?
[Admin] Manage list user, active, block user

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by hoangtrungthong** [comment]
ready

File: resources/views/admin/dashboard.blade.php [hunk 1]
```diff
@@ -1,8 +1,14 @@
 <x-app-layout>
     <x-slot name="header">
-        <h2 class="font-semibold text-xl text-gray-800 leading-tight">
-            {{ __('Dashboard') }}
-        </h2>
+        <nav class="flex items-center justify-between flex-wrap bg-teal-500 p-6">
+            <div class="w-full block flex-grow lg:flex lg:items-center lg:w-auto">
+              <div class="text-sm lg:flex-grow">
+                <a href="{{ route('admin.manageUser') }}" class="block mt-4 lg:inline-block lg:mt-0 text-teal-200 hover:text-blue-800 mr-4">
+                  {{ __('Manage User') }}
```
Comment (ghost) [inline resources/views/admin/dashboard.blade.php]: dùng key cho bản dịch thay vì text nhé

File: resources/views/admin/users/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<x-app-layout>
+    <x-slot name="header">
+        <h2 class="font-semibold text-xl text-gray-800 leading-tight py-6 px-4 sm:px-6 lg:px-8">
+            {{ __('Manage User') }}
```
Comment (ghost) [inline resources/views/admin/users/index.blade.php]: tương tự

**Comment by hoangtrungthong** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**